docker/README.md

```diff
@@ -0,0 +1,104 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# What is this?
+This is a docker image to do a POC with Yosegi.
+
+![docker images](images/images.png)
+
+# Create docker images
+
+Please execute the command in this directory.
+
+```
+sh build.sh
+```
+
+# Run docker images
+
+Launch the image with docker-compose.
+
+```
+docker-compose up -d
+```
+
+# Run hive
+
+Ser's results are sent to logger_server from datagen's docker image
```

Comment:
I don't understand "Ser's", could you please check spell miss or describe the "Ser's".

Reply:
Thank you.
I fixed it.

Reply:
Thank you for your consideration. I check the fixation later and approve this PR.